girder/molecules/server/cclib.py

```diff
@@ -0,0 +1,138 @@
+import cclib
+import json
+import tempfile
+
+def _cclib_to_cjson_basis(basis):
+    shell_type_map = {
+        's': 0,
+        'p': 1,
+        'd': 2,
+        'f': 3,
+        'g': 4,
+        'h': 5,
+        'i': 6,
+        'k': 7,
+        'l': 8,
+        'm': 9,
+        'n': 10,
+        'o': 11
+    }
```

Comment:
Is this available in cclib?

Reply:
After a quick search I couldn't find it in `cclib`, `openbabel`, `avogadrolibs`